### PR TITLE
Auto-populate hub template variables

### PR DESCRIPTION
**Pull request by yuvipanda**
Removes need to manually 'refresh' hub variables
after each deploy of the dashboard

### CODE REVIEW

**Comment by minrk** [comment]
Nice! I'm getting 403 on the `api("/datasources")` request. Is this perhaps not in the Editor role?

**Comment by yuvipanda** [comment]
@minrk yeah, I don't think editors can do things to datasources. I'll document that we need admin role?

**Comment by minrk** [comment]
Yeah, that makes sense. It'd be nice to not need it, so maybe a comment indicating which step requires the admin role in case we find another way to achieve the goalo?

**Comment by yuvipanda** [comment]
@minrk done!

File: README.md [hunk 1]
```diff
@@ -31,13 +31,14 @@ via code. This can then be deployed on any Grafana instance!
 
 3. A recent version of Grafana, with a prometheus data source already added.
 
-4. An API key with 'editor' permissions. This is per-organization, and you can make a new one
```
Comment (minrk) [inline README.md:34]: I think it's called 'Admin'

Comment (yuvipanda) [reply README.md:34]: @minrk good catch, fixed!

**Comment by minrk** [comment]
This is rad! Only improvement I can think of is to set the initial value as the first item in the list, if that's possible. As it is now, the dashboard starts with a bunch of "No data" since the variable is empty.

**Pull request merged**